executor/profile.go

```diff
@@ -0,0 +1,48 @@
+package executor
+
+import "golang.org/x/sys/unix"
+
+type Config struct {
+	Cpu       unix.Rlimit
+	Aspace    unix.Rlimit
+	Core      unix.Rlimit
+	Stack     unix.Rlimit
+	Fsize     unix.Rlimit
+	Nproc     unix.Rlimit
+	Memory    unix.Rlimit
+	Clock     uint
+	Minuid    int32
+	Maxuid    int32
+	ChrootDir string
+	// ErrorFile string
+	// UsageFile string
+	InFile  string
+	OutFile string
+	Cmd     string
+	Envv    []string
+}
+
+func (conf *Config) Set_defaults() {
```

Comment:
Please change the naming convention of the function here
The convention in Go is to use `FunctionName` or `functionName`
You may refer to [this](https://go.dev/doc/effective_go#names)